### ISSUE TITLE
List and get SSID Does Not Work 

### ISSUE THREAD

**Issue by aminjoharinia** [once10301/wifi#31]
Hi
I am using ANDROID X
getting list is not working, it always returns []
getting SSID returns unknown SSID, and also level not showing anything
What Should I Do?

**Comment by aminjoharinia** [comment]
I Found The Reason
From Android 8.0 onwards we won't be getting SSID of the connected network unless GPS is turned on.

**Comment by DhavalRKansara** [comment]
> I Found The Reason
> From Android 8.0 onwards we won't be getting SSID of the connected network unless GPS is turned on.

Yeah, GPS is a prerequisite for getting wifi information in both Android and iOS.

You have ask for `ACCESS_FINE_LOCATION` and `ACCESS_COARSE_LOCATION` permission first as mentioned in [Android Developer document](https://developer.android.com/guide/topics/connectivity/wifi-scan) only after allowing this permission you'll get the wifi information.

Please close the issues if you find the answer.

**Comment by rimedar** [comment]
Hi, So, the GPS must always be active to use this package? Or I can configure something so that this is not needed? thanks
